packages/nx-forge/src/executors/build/lib/compile-webpack.ts

```diff
@@ -10,7 +10,7 @@ export function compileWebpack(
   const webpackOptions: WebpackExecutorOptions = {
     target: 'node',
     compiler: 'tsc',
-    outputPath: joinPathFragments(options.outputPath, 'src'),
+    outputPath: joinPathFragments('dist', options.sourceRoot),
```

Comment:
We have to be careful here not to mix up the semantic meaning of `options.sourceRoot` with `options.outputPath`. The issue is that `normalize-options.ts` normalizes the relative output path to an absolute one but the downstream webpack executor expects a relative path. I am checking if it's better to change this in `normalize-options.ts`, however, that probably requires updating other parts of the plugin which currently assume an absolute path.